product_docs/docs/pem/9/pem_architecture.mdx

```diff
@@ -42,8 +42,8 @@ The PEM server consists of an instance of Postgres, an instance of the Apache we
 
 The instance of Postgres (a database server) and an instance of the Apache web-server  HTTPD) can be on the same host or on separate hosts.
 
-    !!! Note
-        All the PEM features are available on either backend database server you select: PostgreSQL or EDB Postgres Advanced Server.
+!!! Note
+    All the PEM features are available on whichever backend database server you select: PostgreSQL, EDB Postgres Advanced Server, or EDB Postgres Extended Server.
```

Comment:
We don't fully support PGE as a backend yet as we need to build the sslutils extension for it.

Reply:
reverted back to original wording